CHANGELOG.md

```diff
@@ -23,6 +25,35 @@ Classify the change according to the following categories:
     ### Deprecated
     ### Removed
 
+## Develop LDES
```

Comment:
@hdunham I'm seeing some discrepancies and overlap between this branch and: 
- https://github.com/NREL/REopt.jl/pull/354/files -- for the `ElectricStorage` O&M changes by @lixiangk1. Is that PR or this PR more up-to-date for those changes? 
- https://github.com/NREL/REopt.jl/pull/332 -- for the `ElectricStorage` min, max duration inputs by @atpham88, which is an alternative way to implement the **fixed_duration** changes in this branch. 

I think it makes sense to merge each of those branches above into this PR. What do you think?